k8s/openebs-operator.yaml

```diff
@@ -22,12 +22,12 @@ metadata:
 rules:
 - apiGroups: ["*"]
   resources: ["nodes","nodes/proxy"]
-  verbs: ["get","list","watch","create","update"]
+  verbs: ["get", "list", "watch"]
 - apiGroups: ["*"]
-  resources: ["namespaces","services","pods","deployments", "events", "endpoints"]
+  resources: ["namespaces", "services", "pods", "deployments", "events", "endpoints", "configmaps"]
   verbs: ["*"]
 - apiGroups: ["*"]
-  resources: ["persistentvolumes","persistentvolumeclaims"]
+  resources: ["persistentvolumes","persistentvolumeclaims","storagepoolclaims","disks","storagepools"]
```

Comment:
should this be under the below customresources.. so we know they are all related to CRDs.